backend/src/main/java/de/otto/platform/gitactionboard/config/GithubActionConfig.java

```diff
@@ -28,12 +28,17 @@ public String domainName(@Value("${DOMAIN_NAME:https://api.github.com}") String
     return domainName;
   }
 
-  @Bean(name = "ownerName")
-  public String ownerName(@Value("${REPO_OWNER_NAME}") String ownerName) {
-    if (ownerName.isBlank())
+  @Bean(name = "basicAuthUserDetails")
+  public String basicAuthUserDetails(
+          @Value("${BASIC_AUTH_USER_DETAILS:}") String content,
+          @Value("${BASIC_AUTH_USER_DETAILS_FILE_PATH:}") String filePath
+  ) {
+    if (!content.isBlank() && !filePath.isBlank()) {
       throw new IllegalArgumentException(
-          "REPO_NAMES environment variable is either empty or its not set");
-    return ownerName;
+              "You must not use BASIC_AUTH_USER_DETAILS and BASIC_AUTH_USER_DETAILS_FILE_PATH at the same time!");
+    }
+
+    return content.isBlank() ? filePath : content;
```

Comment:
Instead of **filePath**, return the content of given file path so that this bean can be usable in later stage.